I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 13, in script call
    call select_start
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 932, in script call
    call event_03
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 934, in script call
    call event_05
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 936, in script call
    call event_07
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 942, in script call
    call event_12
  File "game/00_hp_rpy/18_events.rpy", line 2822, in script call
    call day_start
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 945, in script call
    call event_13
  File "game/00_hp_rpy/18_events.rpy", line 2945, in script call
    call day_start
  File "game/00_hp_rpy/the_oddities.rpy", line 165, in script call
    call book_block(11,100)
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 948, in script call
    call event_15
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/the_oddities.rpy", line 165, in script call
    call book_block(11,100)
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/17_books.rpy", line 482, in script call
    call reading_book_block(11)
  File "game/00_hp_rpy/17_books.rpy", line 675, in script call
    call book_done_b_check_fire
  File "game/00_hp_rpy/01_hp.rpy", line 738, in script call
    call screen main_menu_01
  File "renpy/common/00console.rpy", line 604, in script call
    python in _console:
  File "game/00_hp_rpy/16_potions.rpy", line 144, in script
    ">[tmp_str]"
KeyError: u'tmp_str'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 13, in script call
    call select_start
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 932, in script call
    call event_03
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 934, in script call
    call event_05
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 936, in script call
    call event_07
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 942, in script call
    call event_12
  File "game/00_hp_rpy/18_events.rpy", line 2822, in script call
    call day_start
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 945, in script call
    call event_13
  File "game/00_hp_rpy/18_events.rpy", line 2945, in script call
    call day_start
  File "game/00_hp_rpy/the_oddities.rpy", line 165, in script call
    call book_block(11,100)
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 948, in script call
    call event_15
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/the_oddities.rpy", line 165, in script call
    call book_block(11,100)
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/01_hp.rpy", line 923, in script call
    call c_r_night
  File "game/00_hp_rpy/17_books.rpy", line 482, in script call
    call reading_book_block(11)
  File "game/00_hp_rpy/17_books.rpy", line 675, in script call
    call book_done_b_check_fire
  File "game/00_hp_rpy/01_hp.rpy", line 738, in script call
    call screen main_menu_01
  File "renpy/common/00console.rpy", line 604, in script call
    python in _console:
  File "game/00_hp_rpy/16_potions.rpy", line 144, in script
    ">[tmp_str]"
  File "F:\DEV\BT-Silver-Base\renpy\ast.py", line 593, in execute
    renpy.exports.say(who, what, interact=self.interact)
  File "F:\DEV\BT-Silver-Base\renpy\exports.py", line 998, in say
    who(what, interact=interact)
  File "F:\DEV\BT-Silver-Base\renpy\character.py", line 817, in __call__
    what = what_pattern.replace("[what]", sub(what, translate=translate))
  File "F:\DEV\BT-Silver-Base\renpy\character.py", line 799, in sub
    return renpy.substitutions.substitute(s, scope=scope, force=force, translate=translate)[0]
  File "F:\DEV\BT-Silver-Base\renpy\substitutions.py", line 229, in substitute
    s = formatter.vformat(s, (), kwargs)
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/string.py", line 549, in vformat
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/string.py", line 571, in _vformat
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/string.py", line 632, in get_field
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/string.py", line 591, in get_value
KeyError: u'tmp_str'

Windows-post2008Server-6.2.9200
Ren'Py 6.18.2.729
Bitch Trainer 1.03
